aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -985,14 +986,17 @@ module AptosFramework::Stake {
         Coin::destroy_burn_cap<TestCoin>(burn_cap);
 
         // Validator 1 and 2 join the validator set.
-        join_validator_set(&validator_1, validator_1_address);
         join_validator_set(&validator_2, validator_2_address);
+        join_validator_set(&validator_1, validator_1_address);
```

Comment:
Changed the order to make sure the validator set is not sorted by joining order.